instrumentation/runtime-metrics/library/kafka-broker.md

```diff
@@ -0,0 +1,32 @@
+# Kafka Broker Metrics
+
+Here is the list of metrics based on MBeans exposed by Kafka broker. <br /><br />
```

Comment:
is it ok to go ahead and sync with the specification naming? https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/instrumentation/kafka.md

(also see https://github.com/open-telemetry/opentelemetry-specification/pull/2837)

Reply:
Hi @trask,

We did a thorough study on the OTel Semantics of metrics, and tried to follow the conventions as much as possible, during this exercise we identified there were some gaps. These are listed below:
   
**Issues:**
*  There are many frameworks for which the categorization is not done yet (like messaging frameworks) and they don’t fall in any of the existing categories (HTTP, Database, System, Process, Runtime Environment). This definitely gives us a scope to address the issue by adding more categories [here](https://opentelemetry.io/docs/reference/specification/metrics/semantic_conventions/).
* Under the current naming convention, framework name is not a part of the metric name, so it could be confusing for the end-user as frameworks which fall under the same category(http, db) might mix up and could result in duplicate metric names or might even lead to their incorrect usage (could come from the application itself), for instance: 

`http.server.active_requests`

The above metric could be from any HTTP server and closely if we see framework’s like Tomcat and Jetty would both have this metric, and due to this it is possible to have a clash in the naming.

**Proposal:**
* Placing the framework name at the beginning gives better clarity and perspective at a glance besides giving the information about the reported metrics and its related framework. This would resolve the ambiguity in current structuring discussed in point 2 above (under issues).
* There could be a few metrics under same framework which can be further classified in given categories such as HTTP, Database, System, Process, Runtime Environment), for instance :
	
```
wildfly.db.client.connections.usage
wildfly.session.activeSessions
```
As observed above, not all metrics under Wildfly can be categorised under a single category, so it makes sense to have framework name as first part of the metrics and for the ones which could be categorised, we can add the category after the framework name.  

Giving the thought on all these parameters, we have defined the Kafka metrics as you see in the PR.